src/lesson2/Lesson2.java

```diff
@@ -0,0 +1,41 @@
+package lesson2;
+
+public class Lesson2 {
+
+    public static void main(String[] args) {
+
+        // Код для проверки методов
+        System.out.println(isSumFrom10To20(5, 6));
+        checkToPositive(5);
+        System.out.println(isNegative(5));
+        multiplyPrint("hello", 4);
+        System.out.println(isLeapYear(2024));
+
+    }
+
+    private static boolean isSumFrom10To20(int a, int b) {
+        return ((a + b >= 10) && (a + b <= 20)) ? true : false;
```

Comment:
можно результат суммирования вынести в отдельную переменную, чтобы не вычислять его дважды

Reply:
```suggestion
        return ((a + b >= 10) && (a + b <= 20));
```
поскольку логическое выражение само по себе возвращает нам boolean значение
аналогично и для других методов